payloadgenerator/associated_fields.py

```diff
@@ -0,0 +1,33 @@
+from base64 import b64encode;
```

Comment:
It just generates payload1 for a field1 and calculates the payload (payload2 = sha1(payload1)) for field2. Actually, it is just a concept about how to workaround the problem of not having other types of attack like burp, as ZAP natively only implements Sniper mode. 

This script tries to implement the BatteringRam idea, but it actually came out better than that, as the second payload can produce something that is related to payload1 but still different, without the need of creating a previous wordlist of sha1(payload1) like Burp does.